js/Common.js

```diff
@@ -1337,13 +1337,14 @@ CRM.strings = CRM.strings || {};
    * @return string
    */
   var currencyTemplate;
-  CRM.formatMoney = function(value, format) {
+  CRM.formatMoney = function(value, format, onlyNumber) {
```

Comment:
I wouldn't call the php function a shining example of usability. I'll add some more thoughts below.